.github/workflows/test.yml

```diff
@@ -14,17 +14,17 @@ jobs:
     steps:
       - name: 'Check out repository'
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 1
 
       - name: 'Set up Java Development Kit'
-        uses: oracle-actions/setup-java@v1
+        uses: actions/setup-java@v4
         with:
-          website: oracle.com
-          release: 17
+          distribution: 'oracle'
+          java-version: 17
```

Comment:
It would be nice to move to JDK 21 soon.

Reply:
Upgrading to use Java 21 will be done in a separate effort: as it might affect [building and testing](https://github.com/openjdk/apidiff/tree/master/make#building-apidiff).